### ISSUE TITLE
[bug]: Fresh install terminates with "CondaEnvException: Pip failed"

### ISSUE THREAD

**Issue by mcbexx** [invoke-ai/InvokeAI#1410]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

8

### What happened?

Did a fresh install of 2.1 since the upgrade process from 2.02 threw errors. Removed the old conda environment (and all files) and followed the manual Windows install guide from scratch. After cloning successfully, creating the environment terminated with the feedback below.
Is the Visual C++ library actually required? If so, which tools from the installer provided in the Visual Studio Link exactly, since it's offering multiple tools/use cases?

I did not run into this issue with any previous versions of Invoke AI. 
I'd appreciate if feedback could be provided in layman's terms :)

Full install log on request.

Pip subprocess error:
  Running command git clone --filter=blob:none --quiet https://github.com/openai/CLIP.git 'n:\InvokeAI\src\clip'
  Running command git clone --filter=blob:none --quiet https://github.com/CompVis/taming-transformers.git 'n:\InvokeAI\src\taming-transformers'
  Running command git clone --filter=blob:none --quiet https://github.com/Birch-san/k-diffusion.git 'n:\InvokeAI\src\k-diffusion'
  Running command git clone --filter=blob:none --quiet https://github.com/invoke-ai/Real-ESRGAN.git 'n:\InvokeAI\src\realesrgan'
  Running command git clone --filter=blob:none --quiet https://github.com/invoke-ai/GFPGAN.git 'n:\InvokeAI\src\gfpgan'
  Running command git clone --filter=blob:none --quiet https://github.com/invoke-ai/clipseg.git 'n:\InvokeAI\src\clipseg'
  Running command git checkout -b models-rename --track origin/models-rename
  branch 'models-rename' set up to track 'origin/models-rename'.
  Switched to a new branch 'models-rename'
  error: subprocess-exited-with-error

  × Building wheel for psutil (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [38 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-310
      creating build\lib.win-amd64-cpython-310\psutil
      copying psutil\_common.py -> build\lib.win-amd64-cpython-310\psutil
      copying psutil\_compat.py -> build\lib.win-amd64-cpython-310\psutil
      copying psutil\_psaix.py -> build\lib.win-amd64-cpython-310\psutil
      copying psutil\_psbsd.py -> build\lib.win-amd64-cpython-310\psutil
      copying psutil\_pslinux.py -> build\lib.win-amd64-cpython-310\psutil
      copying psutil\_psosx.py -> build\lib.win-amd64-cpython-310\psutil
      copying psutil\_psposix.py -> build\lib.win-amd64-cpython-310\psutil
      copying psutil\_pssunos.py -> build\lib.win-amd64-cpython-310\psutil
      copying psutil\_pswindows.py -> build\lib.win-amd64-cpython-310\psutil
      copying psutil\__init__.py -> build\lib.win-amd64-cpython-310\psutil
      creating build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\runner.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_aix.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_bsd.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_connections.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_contracts.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_linux.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_memleaks.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_misc.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_osx.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_posix.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_process.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_sunos.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_system.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_testutils.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_unicode.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\test_windows.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\__init__.py -> build\lib.win-amd64-cpython-310\psutil\tests
      copying psutil\tests\__main__.py -> build\lib.win-amd64-cpython-310\psutil\tests
      running build_ext
      building 'psutil._psutil_windows' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for psutil
ERROR: Could not build wheels for psutil, which is required to install pyproject.toml-based projects

failed

CondaEnvException: Pip failed

================================

Ignoring the error and continuing the manual installation beyond the pip error by running preload_models.py results
in

(invokeai) N:\InvokeAI>python scripts/preload_models.py
Loading Python libraries...

Traceback (most recent call last):
  File "N:\InvokeAI\scripts\preload_models.py", line 15, in <module>
    from tqdm import tqdm
ModuleNotFoundError: No module named 'tqdm'

**Comment by mcbexx** [comment]
Looks like the C++ install is actually required now and should be added to the installation requirements accordingly.

**Comment by lstein** [comment]
I'm not sure what version you're working with, but try adding the line `--prefer-binaries` at the top of `requirements.txt` and let me know what happens.

**Comment by lstein** [comment]
Or better still, try checking out the branch named `release-candidate-2-1-3`. I've just been torture testing installs on Windows systems and it seems to be reliable.

**Comment by thraxus2000** [comment]
Does this comment mean an installer of some kind is coming? I really miss playing with prompts, and my attempt to upgrade to 2.1 seemed to just ruin everything. No amount of fiddling seems to be able to fix it anymore.

**Comment by Domino9752** [comment]
You don't mention what happened after you entered "pip install tqdm"
I too got that error, and reinstalling tqdm got me past it.

**Comment by psychedelicious** [comment]
@mcbexx Did the suggestions in the comments help?

@thraxus2000 Yes, we have an installer which is performing well on all platforms rolling out in v2.2. It should be released within a week.

**Comment by mcbexx** [comment]
> @mcbexx Did the suggestions in the comments help?

I got it to work by installing the C++ dependency and updated to 2.1.3 with no problems shortly after, so I couldn't reliably test them.

**Comment by psychedelicious** [comment]
Ok, I'll call that resolved then. Glad it's working for you.
